renovate.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": [
+    "config:base",
```

Comment:
Someone from renovate suggested to use group:linters as well so upgrades to linters are in one PR. Could’ve helped when eslint (and associated packages) were upgraded earlier.

Reply:
That's the exact kind of change I'd like to use autosynth for :) If we change it, we should do it across all the repositories.